### ISSUE TITLE
History Tab in phpMyAdmin

### ISSUE THREAD

**Issue by Nilucshan** [phpmyadmin/phpmyadmin#13705]
Hi All,

I am thinking of adding a tab for keep track of past queries made by the user. It is available in the current phpMyAdmin, but however we need to click on the **console** tab which is located at the bottom of **SQL** screen.  

If we create a simple tab for HISTORY which is similar to BROWSE, STRUCTURE, SQL tabs,  it will be more user friendly in my point of view. Any ideas on how to implement this feature (Creating a History tab in the GUI). 

I am a newbie and any instruction will be highly appreciated.

**Comment by Nilucshan** [comment]
I was able to create a control user pma, and created a table for history, bookmarks and etc. The past sql queries are showed fine in the screen but i need to make it more user friendly by creating a tab in the GUI. Can anyone help me on making the button for History tab which links the function of pma_history table and the tab.

**Comment by nijel** [comment]
I'm not sure it adding another place to display history will improve UI - there are already quite a lot of tabs and those are always related to currently selected object (table/database)...

**Comment by ksaurabhsinha** [comment]
@nijel i agree, history is a common stuff and is not just related to the present database or table being showed and so having a tab does not make much sense.
Can we try placing the history link on the `top right section`, just adjacent to the `scroll to top` button

**Comment by svivian** [comment]
I don't think that would be an improvement, in fact it would be slower since the tabs along the top are separate "pages" (or AJAX requests) whereas the Console at the bottom is always there and instant. The Console itself could be improved maybe, the design is a little "rough".

**Comment by nijel** [comment]
Improvements in Console could be indeed improved, but we do not want to duplicate it's functions.
